src/guides/v2.4/release-notes/open-source-2-4-2.md

```diff
@@ -1218,9 +1786,19 @@ We have fixed hundreds of issues in the Magento 2.4.2 core code.
 
 *  The **Add to cart** button on the shared wishlist page now works as expected for anonymous, guest, and users who are not logged in.
 
-## Known issue
+<!-- ENGCOM-8062 -->
+
+*  Magento now displays selected configurable product attribute options in the wishlist page as expected.  [GitHub-24091](https://github.com/magento/magento2/issues/24091),  [GitHub-22503](https://github.com/magento/magento2/issues/22503)
+
+<!-- ENGCOM-7779 -->
+
+*  Implemented `ActionInterface` for \Magento\Wishlist\Controller\Shared\Allcart. [GitHub-29537](https://github.com/magento/magento2/issues/29537)
+
+## Known issues
+
+**Issue**: The `deleteRequisitionListOutput` query returns details about the deleted requisition list instead of the remaining requisition lists. <!--- MC-39894-->
 
-**Issue**: Discounts for a specific payment method persist during checkout for a purchase order even when the buyer changes payment method during final checkout. As a result, customers may receive a discount that they are not entitled to. This occurs because a cart rule for the original payment method is still applied despite the change in payment method. **Workaround**: None. <!-- KB-827 -->
+**Issue**: Shoppers cannot add a configurable product to their cart from a non-default store view in a multi-store deployment. Magento displays this error: `Could not add item to cart. Please check required options and try again`. **Workaround**: None. [GitHub-31660](https://github.com/magento/magento2/issues/31660)<!--- PWA-1298-->
```

Comment:
Is this issue only related to the PWA storefront, or also to the default Magento storefront? If it's only PWA, you'd need to explicitly mention this as to not scare users from upgrading with this possibly horrible bug.

Reply:
@hostep, I am going to remove this known issue description from this PR and add it to the next PR later this week when I can get clarification. (I have other known issue descriptions to add, too.) thanks.